tests/core/pm-module/test_pm_init.py

```diff
@@ -1,24 +1,114 @@
 import pytest
 
-VALID_MANIFEST = {
-    'package_name': 'foo',
-    'manifest_version': '2',
-    'version': '1.0.0',
-}
+from eth_utils import (
+    to_canonical_address,
+)
+
+from web3 import Web3
+
+try:
+    from ethpm import Package  # noqa: E402
+    from ethpm.tools import get_manifest as get_ethpm_manifest
+    from ethpm.exceptions import (
+        InsufficientAssetsError,
+    )
+except ImportError:
+    ethpm_installed = False
```

Comment:
You should use this instaed.

https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency

Reply:
my plan is to get rid of all these now that ethpm is more or less in stable alpha

Reply:
unless you think they should be left in regardless?

Reply:
No, if `ethpm` is part of the core requirements then these should go away.  Just suggesting a better way to do conditional skips on import failures that I learned about recently.